### ISSUE TITLE
Makes less assumptions about PWD

### ISSUE THREAD

**Issue by Miepee** [JohnnyonFlame/gmloader-next#1]
- [ ] Add option to provide path to the game apk
- [ ] Add option to provide path to the lib folder

And whatever else may be needed.

**Comment by JohnnyonFlame** [comment]
Not for consumption yet. :p

**Comment by Miepee** [comment]
No, but it would help with testing if the program wouldnt assume that PWD is $HOME even tho it clearly isnt xP
